src/main/java/org/folio/rest/impl/ProxiesForAPI.java

```diff
@@ -49,21 +46,26 @@ public void postProxiesfor(String lang, ProxiesFor entity,
 
     var postgresClient = PgUtil.postgresClient(vertxContext, okapiHeaders);
 
+    final var failureHandler = new FailureHandler(asyncResultHandler, logger,
+      PostProxiesforResponse::respond500WithTextPlain);
+
     userAndProxyUserComboExists(entity, postgresClient)
-      .onFailure(handleFailure(asyncResultHandler))
       .onSuccess(proxyAlreadyExists -> {
         if (Boolean.TRUE.equals(proxyAlreadyExists)) {
           logger.error("Proxy relationship already exists: {}", entity.getId());
+
           Errors existsError = ValidationHelper.createValidationErrorMessage(
             "proxyFor", entity.getId(), "Proxy relationship already exists");
+
           asyncResultHandler.handle(Future.succeededFuture(
             PostProxiesforResponse.respond422WithApplicationJson(existsError)));
           return;
         }
 
         PgUtil.post(PROXY_FOR_TABLE, entity, okapiHeaders, vertxContext,
-          PostProxiesforResponse.class, asyncResultHandler);
-      });
+        PostProxiesforResponse.class, asyncResultHandler);
```

Comment:
Shouldn't this line be tabbed over?
It is a continuation of the function parameters from line 65.